stats/includes/team.inc

```diff
@@ -573,7 +570,7 @@ function uploadImages($preview, $tid, $type)
             $upload_avatar_dir = "$team_avatars_dir/$avatarID";
             move_uploaded_file($_FILES['teamavatar']['tmp_name'], $upload_avatar_dir);
             if ($preview != 1) {
-                mysqli_query(DPDatabase::get_connection(), "UPDATE user_teams SET avatar='$avatarID' WHERE id = $tid") or die(DPDatabase::log_error());
+                DPDatabase::query("UPDATE user_teams SET avatar='$avatarID' WHERE id = $tid");
```

Comment:
same here for avatarID and iconID and tid on 592.

Reply:
Same - these are all integers and enforced upstream and should be using `sprint()` but I'm not up for the lift to modernize this code.

Reply:
I completely understand.